sparse/tests/test_coo.py

```diff
@@ -1606,3 +1607,43 @@ def test_coo_valerr():
 def test_random_idx_dtype():
     with pytest.raises(ValueError):
         sparse.random((300,), density=0.1, format="coo", idx_dtype=np.int8)
+
+
+def test_html_for_size_zero():
+    arr = sparse.COO.from_numpy(np.array(()))
+    ground_truth = "<table>"
```

Comment:
It'd be ideal if ground truth was a literal string, and you didn't have to construct it here.

Copy-pasting code from the function into the test is usually Bad.